test/main.js

```diff
@@ -458,19 +479,21 @@ describe('gulp-sass -- sync compile', () => {
       'inheritance.scss',
     ];
 
+    if (COMPILER === 'sass') expectedSourcesAfter.push('inheritance.css');
```

Comment:
This makes me wonder if something else is wrong... Dart sass includes the dist file while node-sass doesn't.

Reply:
Source maps are not part of the sass language spec and there are difference between engines. It's likely node-sass is wrong in this case. A more resilient approach might be to check for the intersection of the sources rather than a matching setting.

Reply:
Feel free to push your suggestion if you want otherwise let's land it as is.